.azure-pipelines.yml

```diff
@@ -13,8 +13,6 @@ variables:
   MAX_XCODE_VERSION: 11.4_beta
   MAX_PLATFORM_VERSION: 13.4
   MAX_IPHONE_DEVICE_NAME: iPhone 11 Pro Max
-  MAX_TV_XCODE_VERSION: 11.3
-  MAX_TV_PLATFORM_VERSION: 13.3
```

Comment:
mm, what about `Apple TV` as device name and xcode version to `11.4` on `macOS-10.15`?

https://github.com/appium/ruby_lib_core/blob/master/ci-jobs/functional_test.yml
https://github.com/appium/ruby_lib_core/blob/master/test/test_helper.rb#L176